### ISSUE TITLE
Light Version

### ISSUE THREAD

**Issue by stdavis** [tjkohli/nebula-ui#15]
I have really enjoyed this theme! Thanks for all of your work!

Just suggesting that a light version would be great. I like to switch between light and dark depending on the time of day.

**Comment by tjkohli** [comment]
I've been considering this idea, but it's interesting you say you like to switch with the time of day. Is sunset/sunrise usually when you switch?

Perhaps we can have an option to auto-switch the theme based on the time.

**Comment by stdavis** [comment]
Yes, generally I use Light during the daylight and Dark in the evenings. My desk faces some large windows. Auto switching based on sunrise/set would be ideal for me. Time of day would be less useful.

**Comment by tjkohli** [comment]
What's your ideal place for a toggle command? In the Command Palette? Or is the theme configuration fine for you?

**Comment by stdavis** [comment]
Ideal would be the command pallet. But theme config is what I currently do.

**Comment by tjkohli** [comment]
How about automatically changing it at Sunrise/Sunset in your current time zone? Or would that not be useful?

**Comment by stdavis** [comment]
That would be awesome!

**Comment by tjkohli** [comment]
No promises but I'll try my best :)

**Comment by stdavis** [comment]
Thanks for the recommendation on the sunset package. That's cool.

**Comment by tjkohli** [comment]
@stdavis Hey, how is the theme holding up for you? There have been some new updates lately. Do you still want a light theme?

**Comment by stdavis** [comment]
Hey @tjkohli. To be honest I hadn't looked at it in the while. I find myself rotating between a few different themes for variety. I took a look at your latest updates and it looks great!

Yes, I would still be interested in a light version of this theme. I now use sunset (thanks for the tip!) to switch based on sunset/rise to switch between light and dark themes.

**Comment by tjkohli** [comment]
Which light and dark themes do you use and why? I love hearing user feedback so I can improve Nebula.

**Comment by stdavis** [comment]
I use one-light for my light theme. I like it because it's simple and the colors pop nicely for JS.

I used to use nebula for my dark theme but recently switch to one-dark. Only because of the new feature of automatically hiding the tab. It's jarring to have the tab hiding and showing every time I make changes to a file then save it. I don't like the fact that the line that I'm typing on moves up and down.

For example:
![ywetteunii](https://user-images.githubusercontent.com/1326248/32175447-a1f43eee-bd4a-11e7-93f5-0fedd68b05fd.gif)

I wish that there was a config that I could turn this feature off with. Otherwise, I've really enjoyed nebula.

**Comment by tjkohli** [comment]
@stdavis I was actually working on a preference for that feature. Please keep in mind that the up/down animation only happens if you have exactly one tab open. If you can hang in, it'll be released soon!

**Comment by stdavis** [comment]
Just upgraded and unchecked the box. Thanks!
